docs/new-contributors.md

```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Let's start with the relationship between the two important components:
+
+*   **Kubernetes** is a container orchestration system, and is used to run containers on a cluster
+*   **KubeVirt** is an add-on which is installed on-top of Kubernetes, to be able to add basic virtualization functionality to Kubernetes.
```

Comment:
Where reasonable, try to keep line length below 80 chars. A few more cases below, but don't artificially break hyperlinks in the middle